<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectCategoryLogView</class>
 <widget class="QDialog" name="SelectCategoryLogView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="wgtSelectSearchParam" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,0">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="margin">
       <number>4</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="margin">
         <number>4</number>
        </property>
        <item>
         <widget class="QCheckBox" name="chbxSelectValidatorJam">
          <property name="text">
           <string>Замятие купюры</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbxSelectMoneyOut">
          <property name="text">
           <string>Купюра мимо</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbxSelectERROR">
          <property name="text">
           <string>ERROR</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbxSelectPayDaemon">
          <property name="text">
           <string>Данные о платежах</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbxSelectStatusAso">
          <property name="text">
           <string>Данные о статусе терминала</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbxSelectStatusPrinter">
          <property name="text">
           <string>Статусы принтера</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbxSelectStatusValidator">
          <property name="text">
           <string>Статусы Валидатора</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbxSelectConnectionState">
          <property name="text">
           <string>Состояние соединения</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chbxSelectUpdater">
          <property name="text">
           <string>Закачка файлов с сервера</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="btnCloseSelectDataS">
          <property name="text">
           <string>Отмена</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnShowSelectData">
          <property name="text">
           <string>Показать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnDeSelectDataAllS">
          <property name="text">
           <string>Снять всё</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectDataAllS">
          <property name="text">
           <string>Выбрать всё</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnCloseSelectDataS</sender>
   <signal>clicked()</signal>
   <receiver>SelectCategoryLogView</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>357</x>
     <y>413</y>
    </hint>
    <hint type="destinationlabel">
     <x>395</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
